### PR TITLE
.NET Security Analyzer -> .NET Framework Analyzer

### LINKED ISSUE #15632: Title of Page - The .NET Framework Analyzer or The .NET Security Analyzers
I apperciate the work that goes into this documentation. I know it is challenging to keep updated. :smile:

It seems like this page is broadly about _The .NET Framework Analyzer_, but the HTML \<title\> is _The .NET Security Analyzers - .NET | Microsoft Docs_, and it also reads:

> The .NET Security Analyzers must be installed as a NuGet package on every project where you want them to run. Only one developer needs to add them to the project. The analyzer package is a project dependency and will run on every developer's machine once it has the updated solution.

Where is this package?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 171973c7-c225-a776-793f-cf67869ea0b8
* Version Independent ID: f26f0129-8073-cc7e-1b3a-0002ad3af932
* Content: [The .NET Security Analyzers - .NET](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/framework-analyzer)
* Content Source: [docs/standard/analyzers/framework-analyzer.md](https://github.com/dotnet/docs/blob/master/docs/standard/analyzers/framework-analyzer.md)
* Product: **dotnet**
* Technology: **dotnet-standard**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #15632

### CODE REVIEW

File: docs/standard/analyzers/framework-analyzer.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: The .NET Security Analyzers - .NET
-description: Learn how to use the .NET Security Analyzers in the .NET Framework Analyzers package to find and address security risks
+title: The .NET Framework Analyzers - .NET
```
Comment (mairaw) [inline docs/standard/analyzers/framework-analyzer.md]:
```suggestion
title: .NET Framework Analyzers - .NET
```

**Review by mairaw — COMMENTED** [review]
Left one suggestion but I'd like to ping to review this one because he originally wrote this article with the two different titles so not sure if there was any intention behind that.

**Comment by mairaw** [comment]
We might also have to make further updates and keep the title as security analyzer or FxCop analyzers since the https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/ package now also contains a Core version.

**Comment by BillWagner** [comment]
Adding @mikadumont for the correct name of this set of packages going forward.

**Review by BillWagner — APPROVED** [review]
I got confirmation that this is the correct naming going forward. I'll :shipit: now.

Thanks again @Youssef1313

**Comment by BillWagner** [comment]
I'm going to close and reopen to get a fresh build.

**Pull request merged**